sqlglot/dialects/duckdb.py

```diff
@@ -666,6 +666,7 @@ class Generator(generator.Generator):
                 exp.cast(e.expression, exp.DataType.Type.TIMESTAMP),
                 exp.cast(e.this, exp.DataType.Type.TIMESTAMP),
             ),
+            exp.Unhex: rename_func("FROM_HEX"),
```

Comment:
This doesn't seem necessary, `FROM_HEX` and `UNHEX` are [aliases](https://github.com/duckdb/duckdb/blob/981c7a8573b254f8efd6b71f764c5c3bd924a53d/extension/core_functions/scalar/string/functions.json#L276-L281).
```suggestion
```

Reply:
Without this the test I added fails. Note that bigquery and presto also define this conversion.

Reply:
DuckDB already generates `UNHEX` for `MD5Digest`, so generating `FROM_HEX` here is going to look inconsistent.

Given they're aliases, let's remove this entry and fix the test so that it uses `UNHEX` instead.

Reply:
Cool, didn't know that. I tested it with `UNHEX` and works just like with `FROM_HEX`, so I reverted this change.